cmd/node/config/enableEpochs.toml

```diff
@@ -209,7 +209,7 @@
     CheckExecuteOnReadOnlyEnableEpoch = 4
 
     # FreezeAccountFeatureEnableEpoch represents the epoch when the freeze account feature is enabled in the protocol
-    FreezeAccountFeatureEnableEpoch = 10
+    FreezeAccountFeatureEnableEpoch = 1
```

Comment:
should this be modified ?

Reply:
I will change it to 2, so we can better test also what happens before activation.
After compression of flags, it can be changed to 1 (only here, for mainnet config repo it will receive an activation epoch in the future).